pkg/test/integration/remote.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2019 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"os"
+	"path/filepath"
+
+	getter "github.com/hashicorp/go-getter"
+)
+
+// Downloads the given url into a subdirectory in the given path.
+func downloadPath(url, path string) (string, error) {
+	// Pwd is necessary for downloading local files via relative path.
+	pwd, err := os.Getwd()
+	if err != nil {
+		return "", err
+	}
+
+	// Subdirectory is given name of md5 hash of url.
```

Comment:
Does this _have_ to be an md5 hash of the URL, or is this just a convention we've established? If the latter I'd suggest using https://golang.org/pkg/hash/fnv/ rather than MD5, presuming this does not need to be a cryptographically sound hash.

Reply:
Nope! Was actually converting this to `fnv` when you commented :)